app/javascript/packs/leafletMap.js

```diff
@@ -242,3 +247,29 @@ function recalculateTooltipVisibility() {
         floors[currentFloor].rooms.addTo(map);
     }
 }
+
+// Returns a modified version of `f` that only runs if the last call is longer
+// than `limit` ms ago.
+export function ratelimit(f, limit) {
```

Comment:
why are those functions in this file? They don't have anything to do with the map!

Reply:
I suppose you are right, should there be a `util.js`?